### PR TITLE
Add interface AddTask to control groups.

### PR DESCRIPTION
**Pull request by jingxiaolu**
So that we can set tasks when we need.

Signed-off-by: Lu Jingxiao <lujingxiao@huawei.com>

Closes: #49 

Thanks everyone  :smiley:

### CODE REVIEW

**Comment by crosbymichael** [comment]
LGTM

**Comment by jingxiaolu** [comment]
Thanks a lot @crosbymichael :smiley:  
So one more reviewer is needed now?

File: control.go [hunk 1]
```diff
@@ -48,8 +49,10 @@ type Process struct {
 type Cgroup interface {
 	// New creates a new cgroup under the calling cgroup
 	New(string, *specs.LinuxResources) (Cgroup, error)
-	// Add adds a process to the cgroup
+	// Add adds a process to the cgroup (cgroup.procs)
 	Add(Process) error
+	// Add adds a process to the cgroup (tasks)
```
Comment (estesp) [inline control.go]: The comment should be `AddTask adds a process..`; we don't have the linter running here, but that would be required for the comment to match the function name

Comment (jingxiaolu) [reply control.go]: Thanks @estesp :+1: , code updated :smile: , please help to check again~

**Pull request merged**